### PR TITLE
Created testes for RemoteGetAllImagesRequestHandler

### PR DESCRIPTION
**Pull request by llvieira**

### CODE REVIEW

File: src/test/java/org/fogbowcloud/manager/core/intercomponent/xmpp/handlers/RemoteGetAllImagesRequestHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        attributes.put("user-name", "fogbow");
+
+        FederationUser federationUser = new FederationUser("fake-id", attributes);
+
+        this.order = new ComputeOrder();
+        order.setProvidingMember("providingmember");
+        order.setFederationUser(federationUser);
+    }
+
+    // test case: When call the handle method passing a valid IQ object, it must create an OK result IQ and return it.
```
Comment (thiagomanel) [inline src/test/java/org/fogbowcloud/manager/core/intercomponent/xmpp/handlers/RemoteGetAllImagesRequestHandlerTest.java]: "when we ..."

Comment (thiagomanel) [reply src/test/java/org/fogbowcloud/manager/core/intercomponent/xmpp/handlers/RemoteGetAllImagesRequestHandlerTest.java]: ... when the handle method is called ..

File: src/test/java/org/fogbowcloud/manager/core/intercomponent/xmpp/handlers/RemoteGetAllImagesRequestHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        IQ result = this.remoteGetAllImagesRequestHandler.handle(iq);
+
+        //verify
+        Mockito.verify(this.remoteFacade, Mockito.times(1)).getAllImages(
+                Mockito.anyString(), Mockito.any(FederationUser.class));
+
+        String iqId = iq.getID();
+        String orderProvidingMember = order.getProvidingMember();
+        String expected = String.format(TAG_RESULT_ERRO, iqId, orderProvidingMember);
+        Assert.assertEquals(expected, result.toString());
```
Comment (thiagomanel) [inline src/test/java/org/fogbowcloud/manager/core/intercomponent/xmpp/handlers/RemoteGetAllImagesRequestHandlerTest.java]: I miss some exception info in the response

**Comment by llvieira** [comment]
@thiagomanel, I would really appreciate if you could review this pull request again.

**Pull request merged**